adapter-rest-dotnet-client/pom.xml

```diff
@@ -0,0 +1,165 @@
+                        </goals>
+                        <configuration>
+                            <inputSpec>/io/github/fileanalysissuite/adapters/rest/contract/swagger.yaml</inputSpec>
+                            <generatorName>csharp-netcore</generatorName>
+                            <gitUserId>FileAnalysisSuite</gitUserId>
+                            <gitRepoId>adapter-rest-contract</gitRepoId>
+                            <addCompileSourceRoot>false</addCompileSourceRoot>
+                            <packageName>MicroFocus.FAS.Adapters.Rest.Client</packageName>
+                            <configOptions>
+                                <packageVersion>${project.version}</packageVersion>
+                                <packageGuid>{5CC863B4-F7B2-4565-909C-32E8C05B13E9}</packageGuid>
+                                <licenseId>Apache-2.0</licenseId>
+                                <releaseNote>See https://github.com/FileAnalysisSuite/adapter-rest-contract/releases</releaseNote>
+                                <targetFramework>netstandard2.0</targetFramework>
+                            </configOptions>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
+                        <VERSION>${project.version}</VERSION>
+                    </environmentVariables>
+                    <workingDirectory>${project.build.directory}/generated-sources/openapi</workingDirectory>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>exec-dotnet-restore</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>dotnet</executable>
+                            <arguments>
+                                <argument>restore</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>exec-dotnet-build</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>dotnet</executable>
+                            <arguments>
+                                <argument>build</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <!-- <execution>
```

Comment:
Remove the test section instead of commenting it out.

Reply:
Removed